src/NexusMods.App.UI/WorkspaceSystem/Panel/IPanelViewModel.cs

```diff
@@ -0,0 +1,35 @@
+using Avalonia;
+
+namespace NexusMods.App.UI.WorkspaceSystem;
+
+public interface IPanelViewModel : IViewModelInterface
```

Comment:
I would add a comment describing what a panel is in this context and its purpose.

Reply:
I'll add documentation once I add tabs to the system.